tests/test_utils.py

```diff
@@ -0,0 +1,39 @@
+import os
+from tempfile import TemporaryDirectory
+import numpy as np
+from PIL import Image
+from autowebcompat import utils
+
+
+def test_get_bugs():
+    bugs = utils.get_bugs()
+    assert(isinstance(bugs, list))
+
+
+def test_mkdir():
+    d = TemporaryDirectory()
+    direc_path = d.name + "/test"
+    utils.mkdir(direc_path)
+    assert(os.path.isdir(direc_path))
+    utils.mkdir(direc_path)
+
+
+def test_load_image(tmpdir):
+    d = TemporaryDirectory()
+    img = Image.new("RGB", (30, 30))
+    img.save(d.name + "/Image.jpg")
+    img = utils.load_image("Image.jpg", d.name)
+    assert(isinstance(img, np.ndarray))
+
+
+def test_read_labels():
+    labels = utils.read_labels(file_name='labels.csv')
```

Comment:
Create a fake labels.csv file in a temporary directory, so you can assert more things about the result.

Reply:
Actually, don't do it, we'll do it as part of the other test that writes and reads labels.